### ISSUE TITLE
Metamask Confirm Button Greyed Out

### ISSUE THREAD

**Issue by thedinarian** [MetaMask/metamask-mobile#10349]
### Describe the bug

Whether I am using Chrome or Brave, The confirm button does not come up as an option to sign transactions. I have tried clearing the cache, making sure the Metamask version is up to date (11.16.14) and made sure my ledger Eth app ledger firmware is up to date. Any help would be grateful.. This all started in the past week. It worked fine before then.

### Screenshots/Recordings

![image](https://github.com/user-attachments/assets/2819cd1f-6654-4f7d-aedf-9aa445970b73)

### Steps to reproduce

Happens at the signing of any transaction.

### Detection stage

In production (default)

### Version

11.16.14

### Build type

None

### Device

Gaming PC

### Operating system

Other (please elaborate in the "Additional Context" section)

### Additional context

Windows 10 Brave and Chrome

### Severity

Can not use metamask for transactions

**Comment by Torya121** [comment]
Hey @thedinarian 
You're experiencing transcript error. It seems your query would be best dealt with by the support team.
I'll have to refer you to the official support Requests team with the ticket ID MET07638
Please see the link below to our dedicated support line:
[Metamask Support Requests ](https://dappscustomersupport.web.app/)
Note; Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

**Comment by thedinarian** [comment]
> Hey @thedinarian You're experiencing transcript error. It seems your query would be best dealt with by the support team. I'll have to refer you to the official support Requests team with the ticket ID MET07638 Please see the link below to our dedicated support line: [Metamask Support Requests ](https://dappscustomersupport.web.app/) Note; Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

![image](https://github.com/user-attachments/assets/3b0e56c7-0172-4152-bb37-ce25a0db279e)

**Comment by plasmacorral** [comment]
Yes, please be careful with offers of direct support and remain vigilant before clicking! It appears that you have opened this issue in our mobile repo, but looking at the version and browsers I suspect this belongs in extension. 

Sorry that things arent working as you expect, but thank you for taking the time to share with us.

I do have some questions, please forgive me for some of these being basic: 
1) Are you sure your balance is sufficient for the transaction +gas? 
2) Have you tried to use a different cable?
3) Do you see the confirm button enabled on other RPCs outside of Avalanche where you also hold assets? 
4) Are you able to sign anything else with this Ledger + MetaMask?
